app/src/main/res/layout/loyalty_card_view_layout.xml

```diff
@@ -264,6 +264,18 @@
             android:tooltipText="@string/nextCard"
             android:visibility="gone" />
 
+        <ImageButton
+            android:id="@+id/button_update_balance"
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:layout_gravity="end"
+            android:background="@android:color/transparent"
+            android:paddingStart="12dp"
+            android:paddingEnd="12dp"
+            android:src="@drawable/ic_basket_plus"
```

Comment:
Where did you find this icon? I can't find it in Android Studio's icon picker or on https://fonts.google.com/icons. While it doesn't look bad, the design does feel different than other official Android icons.

I'm wondering if maybe [Shopping Cart](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Ashopping_cart%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048) or [Redeem](https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Aredeem%3AFILL%400%3Bwght%40400%3BGRAD%400%3Bopsz%4048) feels better.

Reply:
Noted, I'll switch to the cart. I mocked it up myself in inkscape.